### PR TITLE
Pro-1561 simple tags relationship

### PR DESCRIPTION
**Pull request by stuartromanek**
Goal:
In the media editor's skinny utility editor the relationship field's UI is often squished and looks broken. Simplify this to a Browse button with tag slats listed below it.

- Add new modifier styles for removing the autocomplete search input in relationship inputs
- Fields can now have their own `modifiers` prop. This gets combined with the outer schema's modifiers for fine grained rendering control of fields
- AposButton's `block` modifier was pretty login-button-specific. Remove these rules and make `block` generic
- Add specific styles to the login button at the component level

### CODE REVIEW

File: modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue [hunk 1]
```diff
@@ -38,6 +38,9 @@
                   label="apostrophe:login"
                   button-type="submit"
                   :modifiers="['gradient-on-hover', 'block']"
+                  :attrs="{
+                    'style': 'height: 47px'
+                  }"
```
Comment (stuartromanek) [inline modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue:43]: unrelated to goal, add login-specific styles right into the component.

File: modules/@apostrophecms/ui/ui/apos/components/AposButton.vue [hunk 2]
```diff
@@ -395,8 +395,6 @@ export default {
     box-sizing: border-box;
     display: block;
     width: 100%;
-    height: 47px;
-    max-width: 400px;
```
Comment (stuartromanek) [inline modules/@apostrophecms/ui/ui/apos/components/AposButton.vue:399]: unrelated cleanup

File: modules/@apostrophecms/image/index.js [hunk 3]
```diff
@@ -62,7 +62,8 @@ module.exports = {
       _tags: {
         type: 'relationship',
         label: 'apostrophe:tags',
-        withType: '@apostrophecms/image-tag'
+        withType: '@apostrophecms/image-tag',
+        modifiers: [ 'no-search' ]
```
Comment (boutell) [inline modules/@apostrophecms/image/index.js:66]: Cool, we had this in A2 also.

File: modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue (hunk 1 above)
Comment (abea) [reply modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue:43]: This should be done in the style block of the component.

**Review by abea — CHANGES_REQUESTED** [review]

**Review by abea — APPROVED** [review]

**Review by abea — APPROVED** [review]

**Review by boutell — APPROVED** [review]
